catalog/dags/elasticsearch_cluster/point_alias/point_alias_dag.py

```diff
@@ -0,0 +1,92 @@
+"""
+# Point Alias DAG
```

Comment:
Also, this is mostly a nitpick, but can the DAGs' names include the "es" from Elasticsearch? I know we know this is all about ES, but it feels weird not to have that in the file names!

Reply:
Sure! It is _tagged_ `elasticsearch`, but that can be easy to miss. Easy thing to add in, done!